pipeline/merger/src/test/scala/weco/pipeline/merger/rules/OtherIdentifiersRuleTest.scala

```diff
@@ -82,6 +84,23 @@ class OtherIdentifiersRuleTest
       )
     )
 
+  it("merges METS, Miro, Calm and Sierra source IDs into Tei target") {
```

Comment:
Are these tests replications of the scenarios above?

Reply:
fwiw - we do err on the side of testing behaviour both in the rule tests and in the scenario tests, because historically stuff has gone wrong "in between" and it's been hard to tell which part has broken